motoman_sda10f_moveit_config/launch/move_group.launch

```diff
@@ -13,7 +13,7 @@
   <arg     if="$(arg info)" name="command_args" value="--debug" />
 
   <!-- move_group settings -->
-  <arg name="allow_trajectory_execution" default="true"/> 
+  <arg name="allow_trajectory_execution" default="true"/>
```

Comment:
Just noticed this whitespace change. Could you revert the changes to this file? They are out-of-scope for this PR.